### DISCUSSION TITLE
Update Gravatar CDN

### DISCUSSION THREAD

**Discussion by swwind** [hitwhoj/hitwhoj-legacy#1] [General]
Since <https://cn.gravatar.com> joined the GFW list, we need to change those links to another website (or cdn).

It seems that <https://gravatar.loli.net> was the most popular mirror of gravatar in most Chinese websites, which provides nice network speed and stablity at the same time in most cases. What about changing all of gravatar links to it?

**Comment by zwongeer** [comment]
`Your suggestion is our command!`👀

**Reply by swwind** [reply]
`/kill @e`
